tasks/install.yml

```diff
@@ -0,0 +1,79 @@
+---
+- name: Add Apt Key for NodeJS PPA
+  apt_key:
+    id: 1655A0AB68576280
+    keyserver: keyring.debian.org
+
+- name: Add NodeJS 14 PPA
+  apt_repository:
+    repo: "deb https://deb.nodesource.com/node_14.x {{ ansible_distribution_release }} main"
+    filename: nodesource
+
+- name: Install npm
+  apt:
+    name:
+      - nodejs
+
+- name: Add build user
+  user:
+    name: build-user
+
+- name: Download archive
+  get_url:
+    url: "https://github.com/Johni0702/mumble-web/archive/{{ version }}.tar.gz"
+    dest: "{{ archive_file }}"
+    checksum: "{{ checksum }}"
+  become: False
+  delegate_to: localhost
+
+- name: Clean and create build directory
```

Comment:
I'm personally not a fan of tasks that are always `changed`. You could however `register` the previous task and only execute this task if the previous one was changed.

Reply:
But this whole file is only included if a recompile is necessary so this is only changed when it actually needs to be done. Or am I missing your point?

Reply:
Yes, I saw when going through the rest of the files ;)